services/director-v2/src/simcore_service_director_v2/modules/dynamic_services.py

```diff
@@ -25,7 +25,7 @@ def setup(app: FastAPI, settings: DynamicServicesSettings):
     def on_startup() -> None:
         ServicesClient.create(
             app,
-            client=httpx.AsyncClient(),
+            client=httpx.AsyncClient(timeout=ClientRequestSettings().total_timeout),
```

Comment:
this is not the right way to get settings in the code. 
The workflow is:
- application starts and creates an instance of AppSettings (see [init_app](https://github.com/ITISFoundation/osparc-simcore/blob/2834fd9fc33da80ee01b0477068be46bf1892e73/services%2Fdirector-v2%2Fsrc%2Fsimcore_service_director_v2%2Fcore%2Fapplication.py#L33)). This instance is typically initialized with environment variables BUT it is NOT the only way (e.g. dot-env files, contructor variables, secret files). See https://pydantic-docs.helpmanual.io/usage/settings/#field-value-priority.
- the settings are stored as an entry in the app state and shall be that instance that we consult to get any value

In your case, you create an instance yourself which even if most of the time probably will work, but is breaking the policy i describe above.

In this particular function, it is quite easy since you are insdie of ``setup`` that receives an ``app``

Reply:
Some usage of these settings were misleading. Moved to fetch via app instance.